packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -156,7 +158,31 @@ const relativeCreatedAt = computed(() => dayjs(new Date(debug.value.createdAt!))
   Ex: 1 second which is 1000ms = 00m 01s
   Ex: 1 hour and 1 second which is 3601000ms = 01h 00m 01s
 */
-const totalDuration = computed(() => dayjs.duration(debug.value.totalDuration ?? 0).format('HH[h] mm[m] ss[s]').replace(/^0+h /, ''))
+const formatDuration = (duration: number) => {
+  return dayjs.duration(duration).format('HH[h] mm[m] ss[s]').replace(/^0+h /, '')
+}
+
+const formatCreatedAt = (createdAt: string) => {
+  return dayjs(new Date(createdAt)).fromNow()
+}
```

Comment:
Could extract these as composable logic alongside `useDurationFormat` to make them easier to test

Reply:
I decided to just leave these here for now since they are only used here.  I added some tests in the CT tests to cover the scenarios.